OpenRA.Mods.Common/Traits/Render/WithDecoration.cs

```diff
@@ -15,6 +15,8 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	public enum ReferencePoints { TopLeft, BottomRight }
```

Comment:
Well, there's the planned spy insignia, I'm currently working on adding fuel support, which will have a low-on-fuel decoration, similarly one could think of an out-of-ammo decoration and so on. Also you never know what modders might come up with :)